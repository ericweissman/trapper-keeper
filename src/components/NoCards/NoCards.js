import React from 'react'
import { Link } from 'react-router-dom'

export const NoCards = () => {
  return (
    <div className='no-cards'>
      <h3>You currently have no notes</h3>
      <Link to='/new-note'>Start a new note!</Link>
    </div>
  )
}

export default NoCards