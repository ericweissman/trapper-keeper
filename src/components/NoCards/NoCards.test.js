import React from 'react'
import { shallow } from 'enzyme'
import NoCards from './NoCards'

describe('NoCards', () => {
  let wrapper
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<NoCards />)
    expect(wrapper).toMatchSnapshot()
  })
})