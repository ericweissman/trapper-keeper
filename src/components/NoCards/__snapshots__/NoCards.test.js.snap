// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoCards should have match the correct snapshot 1`] = `
<div
  className="no-cards"
>
  <h3>
    You currently have no notes
  </h3>
  <Link
    replace={false}
    to="/new-note"
  >
    Start a new note!
  </Link>
</div>
`;
